src/pages/Pools/hooks/useLiquidityEventChart.ts

```
@@ -27,27 +34,55 @@
+        const dataV3ByDay = dataVolumeV3.find((item) => item.time === dataV2ByDay.time);
+
+        return {
+          time: dataV2ByDay.time,
+          value: dataV2ByDay.value + (dataV3ByDay?.value ?? 0)
+        };
+      });
+
+      setCurrentDataLiquidity(combinedData);
+      if (combinedData.length > 0) {
+        setCurrentItem({ ...combinedData[combinedData.length - 1] });
+        onUpdateCurrentItem && onUpdateCurrentItem(combinedData[combinedData.length - 1]?.value || 0);
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)